README.md

```diff
@@ -51,21 +53,35 @@ The guidelines can be found here (TBD)
 
 ## Usage
 
-##### Development
+### Development
+
+```bash
```

Comment:
`bash` Isn't a supported syntax highlighter. Check https://github.com/github/linguist/blob/master/vendor/README.md for reference.

Reply:
This seem to work in github md. Please see:
https://stackoverflow.com/questions/20303826/highlight-bash-shell-code-in-markdown

Reply:
`bash` Isn't recognised as a highlight identifier. If you remove it, you won't see a difference.